x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -82,13 +83,17 @@ export const ElasticsearchOverview = () => {
     () => (consolePlugin?.EmbeddableConsole ? <consolePlugin.EmbeddableConsole /> : null),
     [consolePlugin]
   );
+  const [selectedPipeline, setSelectedPipeline] = React.useState<string>('');
 
   const codeSnippetArguments: LanguageDefinitionSnippetArguments = {
     url: elasticsearchURL,
     apiKey: clientApiKey,
     cloudId,
+    ingestPipeline: selectedPipeline,
   };
 
+  const { data } = apiService.useLoadPipelines();
```

Comment:
Beware that the `GET /_ingest/pipeline` API returns deprecated pipelines too, you most likely dont want to surface those to the users.

Reply:
Updated